app/models/calculator/trend_percentage.rb

```diff
@@ -1,18 +1,24 @@
 class Calculator::TrendPercentage
-  attr_reader :current_value, :previous_value
+  def self.calculate(*args)
+    new(*args).calculate
+  end
 
-  def initialize(current_value, previous_value)
-    @current_value = current_value
-    @previous_value = previous_value
+  def initialize(current, previous)
+    @current = current
+    @previous = previous
   end
 
-  def run
-    return 0 if previous_value <= 0
+  def calculate
+    return 0 if previous <= 0
 
     trend
   end
 
+private
+
+  attr_reader :current, :previous
```

Comment:
We aren't we just referencing these are instance variables using `@`?

Reply:
Addressed in https://github.com/alphagov/content-data-admin/pull/457/commits/5ad881de50f86420b787384b7200c1815d78c031